app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -79,6 +81,21 @@ To be eligible for a scholarship, you’ll need at least one of the following:
 
 In exceptional circumstances, if you achieved a 2:2 in your degree you may still be awarded a scholarship if you have significant relevant experience.
 
+#### Benefits
+
+Scholarships are provided by charitable organisations with a passion for their subject area.
+
+They offer more than just financial financial benefits, including:
+
+* access to subject specialist events and webinars
+* online and face-to-face workshops across the country
+* mentoring and coaching
+* a community of ex-scholars, teachers and subject leaders
+* access to classroom resources
+* CPD and networking events during your teacher training and beyond
```

Comment:
do people know what `CPD` means @EllieNodder? I think we should probably expand/reword?